packages/frontend/src/pages/QuadletsList.svelte

```diff
@@ -76,7 +76,7 @@ let data: (QuadletInfo & { selected?: boolean })[] = $derived(
     }
 
     if (match && searchTerm.length > 0) {
-      match = quadlet.service?.includes(searchTerm) ?? false;
+      match = quadlet.path.includes(searchTerm) ?? false;
```

Comment:
You need to rebase

Reply:
At the moment this is not merged yet (https://github.com/podman-desktop/extension-podman-quadlet/pull/782)

Reply:
So you can rebase and remove this commit because it is not related to this PR